config/dev_settings.txt

```diff
@@ -77,4 +77,11 @@ IV_ULIMIT_NOFILE_HARD=65535
 IV_ULIMIT_NPROC_SOFT=5000
 IV_ULIMIT_NPROC_HARD=5000
 IV_RES_LIMIT_CPUS=0.25
-IV_RES_LIMIT_MEM=128M
\ No newline at end of file
+IV_RES_LIMIT_MEM=128M
+
+IP_ULIMIT_NOFILE_SOFT=65535
+IP_ULIMIT_NOFILE_HARD=65535
+IP_ULIMIT_NPROC_SOFT=5000
+IP_ULIMIT_NPROC_HARD=5000
+IP_RES_LIMIT_CPUS=0.5
+IP_RES_LIMIT_MEM=128M
```

Comment:
does this work ? with 128GB RAM ? Seems like too little for psql to work with

Reply:
Memory and CPU limit was increased